src/gluonts/ev/metrics.py

```diff
@@ -0,0 +1,238 @@
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class MeanAbsolutePercentageError:
+    forecast_type: str = "0.5"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=partial(
+                absolute_percentage_error, forecast_type=self.forecast_type
+            ),
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class SymmetricMeanAbsolutePercentageError:
+    forecast_type: str = "0.5"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        return StandardMetricEvaluator(
+            map=partial(
+                symmetric_absolute_percentage_error,
+                forecast_type=self.forecast_type,
+            ),
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class MeanScaledIntervalScore:
+    alpha: float = 0.05
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        assert_axis_is_None_or_one(axis, self.__class__.__name__)
+
+        return StandardMetricEvaluator(
+            map=partial(scaled_interval_score, alpha=self.alpha),
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class MeanAbsoluteScaledError:
+    forecast_type: str = "0.5"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        assert_axis_is_None_or_one(axis, self.__class__.__name__)
+
+        return StandardMetricEvaluator(
+            map=partial(
+                absolute_scaled_error, forecast_type=self.forecast_type
+            ),
+            aggregate=Mean(axis=axis),
+        )
+
+
+@dataclass
+class NormalizedDeviation:
+    forecast_type: str = "mean"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        def post_process(
+            abs_error_sum: np.ndarray, abs_label_sum: np.ndarray
+        ) -> np.ndarray:
+            return abs_error_sum / abs_label_sum
+
+        return DerivedMetricEvaluator(
+            metrics={
+                "abs_error_sum": AbsoluteErrorSum(
+                    forecast_type=self.forecast_type
+                )(axis=axis),
+                "abs_label_sum": AbsoluteLabelSum()(axis=axis),
+            },
+            post_process=post_process,
+        )
+
+
+@dataclass
+class RootMeanSquaredError:
+    forecast_type: str = "mean"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        def post_process(mse: np.ndarray) -> np.ndarray:
+            return np.sqrt(mse)
+
+        return DerivedMetricEvaluator(
+            metrics={
+                "mse": MeanSquaredError(forecast_type=self.forecast_type)(
+                    axis=axis
+                )
+            },
+            post_process=post_process,
+        )
+
+
+@dataclass
+class NormalizedRootMeanSquaredError:
+    forecast_type: str = "mean"
+
+    def __call__(self, axis: Optional[int] = None) -> MetricEvaluator:
+        def post_process(
```

Comment:
I think it's better to give them some better name and make them `staticmethod`s.